stable/external-dns/templates/secret.yaml

```diff
@@ -33,7 +33,4 @@ data:
   {{- if .Values.pdns.apiKey }}
   pdns_api_key: {{ .Values.pdns.apiKey | b64enc | quote }}
   {{- end }}
-  {{- range $key, $value := .Values.extraEnv }}
```

Comment:
Why do you delete extraEnv from the secrets?

Reply:
My reasoning is that these are extra environment variables that don't fit in any values of the chart. If the user wants them in a secret they can put them in one and reference them with valueFrom.secretKeyRef. Otherwise, they can provide the value directly.

Reply:
I think that makes sense. Can we get this merged? It is broken as of now.